### PR TITLE
MSBuild: Clarify exec regex behavior

### PR DESCRIPTION
**Pull request by rainersigwald**
Some good doc feedback in https://duerrenberger.dev/blog/2024/06/11/error-handling-with-msbuilds-exec-task/. I was surprised by the behavior too!

### CODE REVIEW

**Comment by Court72** [comment]
@ghogen

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/msbuild/exec-task.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ The following table describes the parameters for the `Exec` task.
...
-|`CustomErrorRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot error lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Default: `null` (no custom processing).|
-|`CustomWarningRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot warning lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Default: `null` (no custom processing).|
+|`CustomErrorRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot error lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Unless `IgnoreStandardErrorWarningFormat` is specified this regular expression is *in addition* to the [standard ones](./msbuild-diagnostic-format-for-tasks.md).<br /><br />Default: `null` (no custom processing).|
```
Comment (ghogen) [inline docs/msbuild/exec-task.md:33]:
```suggestion
|`CustomErrorRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot error lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Unless `IgnoreStandardErrorWarningFormat` is specified, this regular expression is *in addition* to the [standard ones](./msbuild-diagnostic-format-for-tasks.md).<br /><br />Default: `null` (no custom processing).|
```

File: docs/msbuild/exec-task.md [hunk 2]
```diff
@@ -30,12 +30,12 @@ The following table describes the parameters for the `Exec` task.
...
+|`CustomWarningRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot warning lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Unless `IgnoreStandardErrorWarningFormat` is specified this regular expression is *in addition* to the [standard ones](./msbuild-diagnostic-format-for-tasks.md).<br /><br />Default: `null` (no custom processing).|
```
Comment (ghogen) [inline docs/msbuild/exec-task.md:34]:
```suggestion
|`CustomWarningRegularExpression`|Optional `String` parameter.<br /><br /> Specifies a regular expression that is used to spot warning lines in the tool output. This is useful for tools that produce unusually formatted output.<br /><br />Unless `IgnoreStandardErrorWarningFormat` is specified, this regular expression is *in addition* to the [standard ones](./msbuild-diagnostic-format-for-tasks.md).<br /><br />Default: `null` (no custom processing).|
```

**Review by ghogen — APPROVED** [review]

**Comment by ShannonLeavitt** [comment]
@ghogen - If you're ready for us to merge, could you sign off?

**Pull request merged**